### PR TITLE
added CollectStatistics option to rrdcached plugin

### PR DESCRIPTION
**Pull request by michakrause**

### CODE REVIEW

File: templates/plugin/rrdcached.conf.erb [hunk 1]
```diff
@@ -22,4 +22,7 @@
 <% if @xff -%>
   XFF "<%= @xff %>"
 <% end -%>
+<% if @collectstatistics -%>
```
Comment (txaj) [inline templates/plugin/rrdcached.conf.erb]: please use unless like it's done here https://github.com/michakrause/puppet-module-collectd/blob/rrdcached-statistics/templates/plugin/rrdcached.conf.erb#L4-L6

**Comment by txaj** [comment]
Since you are using this option, would you care to help to document it ? https://github.com/collectd/collectd/issues/907 i can make the PR if you don't have time. I'm not sure on how to word it.

**Comment by michakrause** [comment]
Ok, I changed the template to use unless.
I will add a few lines to collectds docs, and create a PR.

**Comment by txaj** [comment]
:+1:

**Comment by blkperl** [comment]
Thanks for the contribution!

**Pull request merged**